docs/common/utilities.ts

```diff
@@ -1,11 +1,13 @@
+import GithubSlugger from 'github-slugger';
+
 /**
  * Converts any object to string accepted by _Slugger_.
  * This is needed, because sometimes we receive pure string node,
  * but sometimes (e.g. when using styled text), we receive whole object (React.Element)
  *
  * @param {any} node React Node object to stringify
  */
-export const toString = node => {
+export const toString = (node: any): string => {
```

Comment:
```suggestion
export const toString = (node: React.NodeObject | string): string => {
```
I've got to admit that this function is confusing, because:
- the docs are saying something
- type is saying sth different 🤔